src/components/atoms/ButtonNG/index.ts

```diff
@@ -0,0 +1 @@
+export { ButtonNG } from "./ButtonNG";
```

Comment:
A bit of a missing piece for `ButtonNG` to align with the rest of the components